aten/src/ATen/native/quantized/cpu/qconcat.cpp

```diff
@@ -31,6 +31,26 @@ bool is_valid_quantization_scheme(const Tensor& t) {
   return (qtype == kPerTensorAffine) || (qtype == kPerTensorSymmetric);
 }
 
+bool all_inputs_sharing_qparams(TensorList qxs) {
```

Comment:
is the performance of this neglibigle?

Reply:
metadata heck should be fast, maybe qparam check for per channel quant will be slower, however I think we only have per tensor quant for activations so it should be OK.

Maybe I should error out for the per channel case